### DISCUSSION TITLE
PACE BMS Modbus Protocol For RS485 V1.3

### DISCUSSION THREAD

**Discussion by bartixmayo** [syssi/esphome-pace-bms#13] [Ideas]
[RS485protocol-battery.pdf](https://github.com/syssi/esphome-pace-bms/files/12117853/RS485protocol-battery.pdf)
I put PACE BMS Modbus Protocol specs that works with Cosuper SPE5000-PR248L inverter. Maybe hepful in some cases.

**Comment by syssi** [comment]
You can find a very similar protocol description at the `docs/` directory of this project.
